src/SDFileLogger.h

```diff
@@ -63,6 +63,10 @@ class SDFileLogger final : public LoggerBase
 		flush();
 	}
 
+	void update_file_name(uint8_t fileNumber){
```

Comment:
Instead of adding a new function that changes the name of the file, why not add it as a parameter in the begin function?  Calling this function after begin would have no impact.

Reply:
Also, this way I believe you can get rid of the filename member variable in this class.